wikibase/README.md

```diff
@@ -9,12 +9,10 @@ Automated build.
 
 ### Tags
 
-Image name                                                            | Parent image
------------------------------------------------------------------     | ------------------------
-`wikibase/wikibase` : `latest`, `1.34`, `latest-base`, `1.34-base`    | [mediawiki:1.34](https://hub.docker.com/_/mediawiki/)
-`wikibase/wikibase` : `1.31`, `lts`, `lts-base`, `1.31-base`          | [mediawiki:1.31](https://hub.docker.com/_/mediawiki/)
-`wikibase/wikibase` : `latest-bundle`, `1.34-bundle`                  | [wikibase:1.34](https://hub.docker.com/r/wikibase/wikibase/)
-`wikibase/wikibase` : `lts-bundle`, `1.31-bundle`                     | [wikibase:1.31](https://hub.docker.com/r/wikibase/wikibase/)
+Image name                                                                               | Parent image
```

Comment:
This PR currently leaves 1.31 and 1.34 being built by travis.
If that is intended then these should remain in the docs here.
If that is not intended then they should be totally removed in this PR.

Looking at https://www.mediawiki.org/wiki/Version_lifecycle I think we want to keep 1.31 and 1.34 for now.
We will likely want to remove 1.34 at the end of this month.
1.31 will remain around until next year.